mne/io/edf/edf.py

```diff
@@ -1597,7 +1882,8 @@ def read_raw_edf(
     Parameters
     ----------
     input_fname : path-like
-        Path to the EDF or EDF+ file.
+        Path to the EDF or EDF+ file or EDF/EDF+ file itself. If a file-like
+        object is provided, preload must be used.
```

Comment:
```suggestion
        object is provided, preload must be used.
        
        .. versionchanged:: 1.10
            Added support for file-like objects.
```
Same in other functions

Reply:
Yeah `read_raw_*` should have them, classes not as critical